freetextresponse/freetextresponse.py

```diff
@@ -361,24 +361,29 @@ def _get_problem_progress(self):
         Returns a statement of progress for the XBlock, which depends
         on the user's current score
         """
-        result = ''
-        if self.score == 0.0:
-            result = ungettext(
-                "{weight} point possible",
-                "{weight} points possible",
-                self.weight,
-            ).format(
-                weight=self.weight
+        if self.weight == 0:
+            result = ''
+        elif self.score == 0.0:
+            result = "({})".format(
+                ungettext(
+                    "{weight} point possible",
+                    "{weight} points possible",
+                    self.weight,
+                ).format(
+                    weight=self.weight
```

Comment:
![comma](http://cjpinard.com/wp-content/uploads/2014/12/keep-calm-and-use-commas-4.png)